hack/.staticcheck_failures

```diff
@@ -1,6 +1,7 @@
 cluster/images/etcd/migrate
 pkg/controller/podautoscaler
 pkg/controller/replicaset
+pkg/kubelet/dockershim
```

Comment:
unrelated change?

Reply:
No the `deprecated` field will cause staticcheck to fail in that file. That's why I excluded it here.